nixos/common/nix.nix

```diff
@@ -7,6 +7,9 @@
   nix.settings.experimental-features = [
     "nix-command"
     "flakes"
+    "repl-flake"
+    "impure-derivations"
+    "auto-allocate-uids"
```

Comment:
for `auto-allocate-uids` to work you need to set `settings.auto-allocate-uids = true;`